dotcom-rendering/src/web/components/Snap.tsx

```diff
@@ -0,0 +1,32 @@
+import { css, Global } from '@emotion/react';
+
+const snapStyles = css`
+	overflow-y: hidden;
+	position: relative;
+`;
+
+type Props = {
+	enriched?: DCREnrichedContent;
+};
+
+export const Snap = ({ enriched }: Props) => {
+	return (
+		<div css={snapStyles}>
+			{enriched?.embedCss !== undefined && (
+				<Global styles={enriched?.embedCss} />
+			)}
+
+			{enriched?.embedHtml !== undefined && (
+				<div
+					dangerouslySetInnerHTML={{ __html: enriched?.embedHtml }}
+				/>
+			)}
+
+			{enriched?.embedJs !== undefined && (
+				<script
```

Comment:
Currently on Frontend the JS is ran at page load, I wonder if we should try and do it differently in DCR and load it asynchronously.